src/components/counter.jsx

```diff
@@ -33,14 +34,15 @@ class Counter extends Component {
 
     return (
       <CounterWrapper>
-        <CounterButton onClick={ requestIncrement }>increment async</CounterButton>
+        <CounterButton onClick={ requestIncrement }>
```

Comment:
Looks like prettier formatting? I'd like to add that in these boilerplates eventually. I'm using it in the basho repo and love it :-)

Reply:
Prettier is the best!